test/bootstrap_form_test.rb

```diff
@@ -228,13 +234,17 @@ class BootstrapFormTest < ActionView::TestCase
           <label class="form-label required" for="user_email">Email</label>
           <input aria-required="true" required="required" class="form-control" id="user_email" name="user[email]" type="email" value="steve@example.com" />
         </div>
-        <div class="form-check">
-          <input #{autocomplete_attr} name="user[terms]" type="hidden" value="0" />
-          <input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" />
-          <label class="form-check-label" for="user_terms">I agree to the terms</label>
+        <div class="mb-3 row">
+          <div class="col-sm-10 offset-sm-2">
+            <div class="form-check">
+              <input #{autocomplete_attr} name="user[terms]" type="hidden" value="0" />
+              <input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" />
+              <label class="form-check-label" for="user_terms">I agree to the terms</label>
+            </div>
+          </div>
         </div>
         <div class="mb-3">
-          <label class="form-label" for="user_misc">Misc</label>
```

Comment:
I wonder why sometimes it wasn't `col-form-label` before?

Reply:
I found a place where the field layout option was not forwarded to the layout label classes method.  Fixed it.